server/app/views/admin/programs/ProgramStatusesView.java

```diff
@@ -226,12 +225,13 @@ private Pair<DivTag, ImmutableList<Modal>> renderStatusItem(
       StatusDefinitions.Status status,
       Form<ProgramStatusesForm> statusEditForm,
       boolean displayOnLoad) {
-    Optional<DomContent> extraContent =
-        status.localizedEmailBodyText().isPresent()
-            ? Optional.of(renderEmailTranslationWarning())
-            : Optional.empty();
     Modal editStatusModal =
-        makeStatusUpdateModal(request, program, statusEditForm, displayOnLoad, extraContent);
+        makeStatusUpdateModal(
+            request,
+            program,
+            statusEditForm,
+            displayOnLoad,
+            status.localizedEmailBodyText().isPresent());
```

Comment:
prefix comment this too for readability as the evaluation doesn't map to the params utility.

Reply:
Done.